plugins/base/src/main/resources/dokka/styles/style.css

```diff
@@ -1038,7 +1056,7 @@ td.content {
 
 @media print, screen and (min-width: 960px) {
     .keyValue {
-        grid-template-columns: 25% 75%;
+        grid-template-columns: 20% 80%;
```

Comment:
Revert to 1.7.10 behaviour. It looks good if `keyValue` table is part of the main content, but first column takes too much space if parameters are within tabbed content (there's not much space as it is)